src/test/java/com/facebook/openwifirrm/modules/ApiServerTest.java

```diff
@@ -68,8 +73,12 @@ public class ApiServerTest {
 	private final Gson gson = new Gson();
 
 	/** Build an endpoint URL. */
-	private String endpoint(String path) {
-		return String.format("http://localhost:%d%s", TEST_PORT, path);
+	private String endpoint(String path, boolean internal) {
```

Comment:
nit: often when you make changes like this, you should add an overloaded method with the original signature to avoid changing every caller:
```java
private String endpoint(String path) {
    return endpoint(path, true);
}
```